src/hooks/useFarmStats.ts

```diff
@@ -27,87 +27,78 @@ function buildFarmStatsQuery(farmAddress: string) {
       totalUnlockedRewards
     }
   }
-`
+`;
 }
 
 type FarmStatsResponse = {
-  apy: BigIntish,
-  tvl: BigIntish,
-  staked: BigIntish,
-  bonusPeriodSec: number,
-  createdTimestamp: number,
-  totalUnlockedRewards: BigIntish
-}
+  apy: BigIntish;
+  tvl: BigIntish;
+  staked: BigIntish;
+  bonusPeriodSec: number;
+  createdTimestamp: number;
+  totalUnlockedRewards: BigIntish;
+};
 
 export function useFarmStats(farmAddress: string) {
-  const [apy, setAPY] = useState(ZERO_BIG_INT)
-  const [tvl, setTVL] = useState(ZERO_BIG_INT)
-  const [totalStaked, setTotalStaked] = useState(ZERO_BIG_INT)
-  const [rewardUnlockRate, setRewardUnlockRate] = useState(ZERO_BIG_INT)
-  const [daysLeft, setDaysLeft] = useState(0)
+  const [apy, setAPY] = useState(ZERO_BIG_INT);
+  const [tvl, setTVL] = useState(ZERO_BIG_INT);
+  const [totalStaked, setTotalStaked] = useState(ZERO_BIG_INT);
+  const [rewardUnlockRate, setRewardUnlockRate] = useState(ZERO_BIG_INT);
+  const [timeRemaining, setTimeRemaining] = useState({});
```

Comment:
Should the default state be an object? Or is using `null` better so we can just detect if the values inside that object actually exist?